Game/SexEngine/FetishHolder.gd

```diff
@@ -33,6 +33,15 @@ func setCharacter(newchar):
 func setFetish(fetishID, interest):
 	fetishMap[fetishID] = interest
 
+func modifyFetishByTier(fetishID, numTiers: int = 1) -> bool:
```

Comment:
I'd call it incrementFetish(). Couldn't understand what the function does just by the name, had to read the code